docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -170,7 +170,7 @@ The component has no special requirements for backing up a database that
 contains encrypted InnoDB tablespaces.
 
 ```{.bash data-prompt="$"}
-$ xtrabackup --backup --target-dir=/data/backup --user=root
+$ xtrabackup --backup --target-dir=/data/backup --user=root --component-keyring-config=/usr/lib64/mysql/plugin/component_keyring_file.cnf
```

Comment:
during the backup, it is not necessary. we detect it by asking the server.

Reply:
please fix the below prepare command.
it shouldn't use --keyring-file-data, instead it should use --component-keyring-config=/usr/lib64/mysql/plugin/component_keyring_file.cnf

Reply:
at line 197 it is